istioctl/pkg/install/verify.go

```diff
@@ -138,13 +138,17 @@ func NewVerifyCommand() *cobra.Command {
 		fileNameFlags = &genericclioptions.FileNameFlags{
 			Filenames: &filenames,
 			Recursive: boolPtr(true),
+			Usage:     "Istio YAML installation file.",
 		}
 	)
 	verifyInstallCmd := &cobra.Command{
 		Use:   "verify-install",
 		Short: "Verifies Istio Installation Status",
 		Long: `
-		verify-install Verifies Istio Installation Status
+		verify-install Verifies Istio Installation Status against the installation file
```

Comment:
change to verifies Istio installation status

Reply:
done